### PR TITLE
fix: reverse transactions

### PR DESCRIPTION
**Pull request by sank8-2**
#3 Have Reversed the payee and paid to in Transaction History Dialog.
New lines 75-77 and 83-85 have the changes.

### CODE REVIEW

File: frontend/src/components/dashboard/BalanceSummary.vue [hunk 1]
```diff
@@ -35,73 +45,86 @@
...
 
 const summaryResource = createResource({
-	url: "batwara.api.get_summary_for_session_user",
-	auto: true
+	url: 'batwara.api.get_summary_for_session_user',
+	auto: true,
 })
 
-const selectedFriend = reactive({name: "friend3@email.com", full_name: "Shivam Ghosh"})
+const selectedFriend = reactive({ name: 'friend3@email.com', full_name: 'Shivam Ghosh' })
```
Comment (NagariaHussain) [inline frontend/src/components/dashboard/BalanceSummary.vue:109]: We have to get rid of this hard coded object.

**Review by NagariaHussain — CHANGES_REQUESTED** [review]

File: frontend/src/components/dashboard/BalanceSummary.vue (hunk 1 above)
Comment (sank8-2) [reply frontend/src/components/dashboard/BalanceSummary.vue:109]: Ok bro,
Can you explain about the error in checking that has happened

**Comment by NagariaHussain** [comment]
You will have to install and run the pre-commit hooks.

File: frontend/src/components/dashboard/BalanceSummary.vue (hunk 1 above)
Comment (NagariaHussain) [reply frontend/src/components/dashboard/BalanceSummary.vue]: Linters are failing

**Pull request closed without merging**